request-schemas.ts

```diff
@@ -0,0 +1,60 @@
+import Joi = require('@hapi/joi');
```

Comment:
rename your files - request/actions/request handlers are to generic. this project can handle another type of data then product

Reply:
and separate your code to folders

Reply:
done